package/gluon-radv-filterd/src/mac.h

```diff
@@ -0,0 +1,8 @@
+#include <stdint.h>
+#include <linux/if_ether.h>
+
+#define F_MAC "%02hhx:%02hhx:%02hhx:%02hhx:%02hhx:%02hhx"
+#define F_MAC_IGN "%*2x:%*2x:%*2x:%*2x:%*2x:%*2x"
+#define F_MAC_VAR(var) var[0], var[1], var[2], var[3], var[4], var[5]
+
+typedef uint8_t macaddr_t[ETH_ALEN];
```

Comment:
The idea is to use a struct
```C
typedef struct {
  uint8_t d[ETH_ALEN];
} macaddr_t;
```
This would allow to simply assign variables of type *macaddr_t* to each other, instead of using memcpy. I don't think it's very important, just how I would implement it.